src/idmodels/sarix.py

```diff
@@ -13,10 +13,27 @@ def __init__(self, model_config):
         self.model_config = model_config
 
     def run(self, run_config):
+        valid_sources = np.array(["nhsn", "nssp"])
+        if ~np.isin(np.array(self.model_config.sources), valid_sources).all():
```

Comment:
I think we should use `not` (logical operation) instead of `~` (bitwise operation). BUT: I might be wrong:

> The exception is when working with specialized libraries like Pandas or NumPy, where ~ is overloaded to perform logical negation on certain data structures